backend/src/main/java/io/debezium/configserver/rest/ConnectorURIs.java

```diff
@@ -20,6 +20,7 @@
     public static final String CONNECTOR_RESUME_ENDPOINT = "/connector/{cluster}/{connectorname}/resume";
     public static final String CONNECTOR_RESTART_ENDPOINT = "/connector/{cluster}/{connectorname}/restart";
     public static final String CONNECTOR_TASK_RESTART_ENDPOINT = "/connector/{cluster}/{connectorname}/task/{tasknumber}/restart";
+    public static final String CONNECTOR_CONFIG_ENDPOINT = "/connectors/{cluster}/{connector-name}/config";
```

Comment:
Could you maybe use the same naming schema like in the other URLs? 
```suggestion
    public static final String CONNECTOR_CONFIG_ENDPOINT = "/connectors/{cluster}/{connectorname}/config";
```